vumi/transports/smpp/config.py

```diff
@@ -27,6 +27,12 @@ class SmppTransportConfig(Transport.CONFIG_CLASS):
         'How long (in seconds) to wait for the SMSC to return with a '
         '`submit_sm_resp`. Defaults to 24 hours.',
         default=(60 * 60 * 24), static=True)
+    disable_ack = ConfigBool(
+        'Disable publishing of `ack` events. In some cases this event '
+        'causes more noise than signal. It can optionally be turned off. '
+        'Defaults to True.',
+        default=False, static=True
```

Comment:
The text and the `default` value contradict each other here. I think the text is wrong.

Also, I don't think the closing paren needs its own line. :-)